polyfill/test/zoneddatetime.mjs

```diff
@@ -730,6 +730,23 @@ describe('ZonedDateTime', () => {
     });
   });
 
+  describe('Maths', () => {
+    const earlier = ZonedDateTime.from('1976-11-18T15:23:30.123456789-03:00[America/Santiago]');
+    const later = ZonedDateTime.from('2019-10-29T10:46:38.271986102-03:00[America/Santiago]');
+    ['years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'].forEach((largestUnit) => {
+      const diff = later.since(earlier, { largestUnit });
+      it(`earlier.since(later, ${largestUnit}) == later.since(earlier, ${largestUnit}).negated()`, () =>
```

Comment:
NOTE: This test won't work for `largestUnit` of `'years'` or `'months'` for some pairs of dates. If it works for this pair, that's just luck. The only guarantee that you can make given our decision of (A) in https://github.com/tc39/proposal-temporal/issues/993#issuecomment-721594580 is that if `d = a.since(b)` then `a.subtract(d).equals(b)`. AFAIK there are no other guarantees that can be made for all pairs of values.

Reply:
What I'm doing for the `DateTime` and `Date` versions of these tests is to run different tests for months/years vs. all other units. And I use pairs of values that trigger the behavior difference.  If you're *not* seeing failures in this test, then I'd suggest changing your earlier/later values to ones where behavior will vary by direction.

Reply:
I have changed this part to the following, which I think should be correct regardless of whether #993 has landed:
```js
    const earlier = ZonedDateTime.from('1976-11-18T15:23:30.123456789-03:00[America/Santiago]');
    const later = ZonedDateTime.from('2019-10-29T10:46:38.271986102-03:00[America/Santiago]');
    // The interchangeability of since() and until() holds for time units only
    ['hours', 'minutes', 'seconds'].forEach((largestUnit) => {
      const diff = later.since(earlier, { largestUnit });
      it(`earlier.since(later, ${largestUnit}) == later.since(earlier, ${largestUnit}).negated()`, () =>
        equal(`${earlier.since(later, { largestUnit })}`, `${diff.negated()}`));
      it(`earlier.until(later, ${largestUnit}) == later.since(earlier, ${largestUnit})`, () =>
        equal(`${earlier.until(later, { largestUnit })}`, `${diff}`));
      it(`${largestUnit} difference symmetrical with regard to negative durations`, () => {
        assert(earlier.subtract(diff.negated()).equals(later));
        assert(later.add(diff.negated()).equals(earlier));
      });
    });
    // For all units, add() undoes until() and subtract() undoes since()
    ['years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'].forEach((largestUnit) => {
      const diff1 = earlier.until(later, { largestUnit });
      const diff2 = later.since(earlier, { largestUnit });
      it(`earlier.add(${diff1}) == later`, () => assert(earlier.add(diff1).equals(later)));
      it(`later.subtract(${diff2}) == earlier`, () => assert(later.subtract(diff2).equals(earlier)));
    });
```
If I have time later, I'll try to add a few test cases that trigger calendar or time zone differences.

Reply:
Looks good. I did a similar split in #1123.